dev/com.ibm.ws.kernel.boot_fat/test-applications/checkJvmAppClasspath/resources/META-INF/permissions.xml

```diff
@@ -17,7 +17,7 @@
     version="7">
 	
    <permission>
-   		<class-name>java.lang.RuntimePermission</class-name>
+   		<class-name>java.security.AllPermission</class-name>
```

Comment:
Why did the permissions need updated for this checkJvmAppClasspath application?

Reply:
The test failed consistently in sandbox runs due a lacking permission. I widened the application's permission set with the concern that we ensure the problem does not surface in other FATs during full regression runs.  We'll know soon.